xds/internal/xdsclient/authority_test.go

```diff
@@ -176,7 +176,7 @@ func (s) TestTimerAndWatchStateOnErrorCallback(t *testing.T) {
 //   - resume mgmt server to accept requests.
 //   - update the mgmt server with resourceB and verifies that the watcher's
 //     update callback is invoked.
-func (s) TestWatchResourceTimerCanRestartOnIgnoredADSRecvError(t *testing.T) {
+func TestWatchResourceTimerCanRestartOnIgnoredADSRecvError(t *testing.T) {
```

Comment:
Hmm ... interesting, why do we need this?

Reply:
Just an error, I brought it back